core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -708,4 +692,45 @@ public void recoverFromStartSync(AlluxioURI uri, long mountId) {
       future.cancel(true);
     }
   }
+
+  @Override
+  public boolean processJournalEntry(JournalEntry entry) {
+    if (entry.hasAddSyncPoint()) {
+      apply(entry.getAddSyncPoint());
+      return true;
+    } else if (entry.hasRemoveSyncPoint()) {
+      apply(entry.getRemoveSyncPoint());
+      return true;
+    } else if (entry.hasActiveSyncTxId()) {
+      File.ActiveSyncTxIdEntry activeSyncTxId = entry.getActiveSyncTxId();
+      setTxId(activeSyncTxId.getMountId(), activeSyncTxId.getTxId());
+      return true;
+    }
+    return false;
+  }
+
+  /**
+   * {@inheritDoc}
+   *
+   * It clears all sync points, and stops the polling thread.
+   */
+  @Override
+  public void resetState() {
+    for (long mountId : mFilterMap.keySet()) {
+      try {
+        // stops sync point under this mount point. Note this clears the sync point and
+        // stops associated polling threads.
+        stopSyncForMount(mountId);
+      } catch (IOException e) {
+        LOG.info("reset: IOException resetting mountId {}, exception {}", mountId, e);
+      } catch (InvalidPathException e) {
+        LOG.info("reset: InvalidPathException resetting mountId {}, exception {}", mountId, e);
```

Comment:
Why prefix with `reset`?

Reply:
removed the prefix (this code was actually just moved)